synapse/handlers/experimental_features.py

```diff
@@ -0,0 +1,55 @@
+# Copyright 2023 The Matrix.org Foundation C.I.C
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+
+from typing import TYPE_CHECKING, Tuple
+
+if TYPE_CHECKING:
+    from synapse.server import HomeServer
+
+
+class ExperimentalFeaturesHandler:
```

Comment:
If this just pipes through to the store methods I'd honestly probably forego it and just call the store directly. 🤷

Reply:
Right on I thought about that but as this is the first time I am adding something like this I followed the form of what else I saw in the codebase. I'll remove this class.